plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManager.java

```diff
@@ -46,6 +46,9 @@
     static final ConfigKey<Integer> s_vmwareOVAPackageTimeout = new ConfigKey<Integer>(Integer.class, "vmware.package.ova.timeout", "Advanced", "3600",
             "Vmware script timeout for ova packaging process", true, ConfigKey.Scope.Global, 1000);
 
+    static final ConfigKey<Boolean> s_vmwareCleanupPortGroups = new ConfigKey<Boolean>("Advanced", Boolean.class, "vmware.cleanup.port.groups",  "true",
+            "Remove unused port groups.", true, ConfigKey.Scope.Global);
```

Comment:
Any reason to keep default value 'true' ? I think, Default value 'false' will keep the old behavior after upgrade, and up to admin to set it to 'true' to cleanup the port groups.

Reply:
Addressed